torchvision/datasets/imagenet.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, root, split='train', download=False, **kwargs):
 
         if download:
```

Comment:
Should the download flag be deprecated?

Reply:
yes, I think so.

Can you do the following:
- make `download=None` by default
- if `download is not None`, raise a deprecation warning